support/pages/accountPage.ts

```diff
@@ -0,0 +1,31 @@
+import { Locator, Page } from '@playwright/test'
+
+export class AccountPage {
+    readonly page: Page
+    readonly accountMenuBtn: Locator
+    readonly accountManageBtn: Locator
+    readonly progressBarSelector: Locator
+    readonly progressBarCurrent: Locator
+    readonly progressBarOption: Locator
+
+    constructor(page: Page) {
+        this.page = page
+        this.accountMenuBtn = this.page.locator('.oc-topbar-avatar')
+        this.accountManageBtn = this.page.locator('#oc-topbar-account-manage')
+
+        this.progressBarSelector = this.page.locator('.account-page-extensions .extension-preference .vs__search')
+        this.progressBarCurrent = this.page.locator('.account-page-extensions .extension-preference .vs__selected span')
+        this.progressBarOption = this.page.getByText('Nyan Cat progress bar')
+    }
+
+    async goToAccountPage() {
+        await this.accountMenuBtn.click()
+        await this.accountManageBtn.click()
+    }
+
+    async selectProgressBarExtension(name: string){
+        await this.progressBarSelector.waitFor()
+        await this.progressBarSelector.click()
+        await this.progressBarOption.click()
+    }
```

Comment:
I have few query regarding progress bar feature. 
- you passed name but i didn't see and usuage?
- I'm not sure but does progress bar extension allowed us to set other progress bar rather than `Nyan Cat progress bar`. If we can choose from multiple type of progress bar  shouldn't there should be  a way which can select different progress bar option using this function .

```suggestion
    async selectProgressBarExtension(name: string){
        await this.progressBarSelector.waitFor()
        await this.progressBarSelector.click()
        await this.progressBarOption.click()
    }
```